tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1619,6 +1619,29 @@ public function updateSpaceQuota(
 		);
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has changed the quota of the "([^"]*)" space to "([^"]*)"$/
```

Comment:
```suggestion
	 * @Given /^user "([^"]*)" has changed the quota of user "([^"]*)" space to "([^"]*)"$/
```

Reply:
Imo `the` is appropriate here because this step can be used in following formats
```
Given user "Admin" has changed the quota of the "Brian Murphy" space to "1"
Given user "Admin" has changed the quota of the "Project" space to "1"
Given user "Admin" has changed the quota of the "Shares Jail" space to "1"
```